### DISCUSSION TITLE
중력을 없애고, 다시 풀면 풀리지 않음

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#1199] [5. Q&A]
zepetoCharacter.MotionV2.Gravity = 0 으로 하고, 캐릭터의 위치를 상단으로 변경한 후 
zepetoCharacter.MotionV2.Gravity = 50 원래값으로 변경하면 중력이 먹지 않고, 공중에 떠 있습니다. 
Scene에서 강제로 내리고, 움직이면 그때부터 원래대로 중력이 먹는데, 자연스럽게 변경하려면 어떻게 하나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, Gravity = 0 으로 했을때 상태 변환이 되지 않는 이슈 재현되어 확인 중입니다.
우선은 0 대신에 0.1 으로 수정해서 사용하시면 다시 중력값이 돌아가므로 값 수정을 권장 드리고 
월드 1.6.0에서 캐릭터 컨트롤러 지면체크 부분이 개선되었기에 월드 업데이트도 권장 드립니다.
감사합니다.
